README.md

```diff
@@ -102,14 +96,16 @@ func main() {
 You will see these logs in your LogDNA dashboard! Make sure to run .Close() when done with using the logger.
 
 ## Tests
-To run a specific test in the provided test suite add your ingestion key under key and run:
+
+Run all tests in the test suite:
+
 ```
-go test -run TestName
+go test ./logger
 ```
 
-To run all the tests in the test suite run:
+Run a specific test:
 ```
-go test -run ''
+go test ./logger -run ^TestLogger_LogWithOptions$
```

Comment:
should be able to just do: go test ./logger -run TestLogger_Log_Levels

Reply:
that doesn't necessarily run only a single test, it would also run `TestLogger_LogWithOptions_foo` if that existed. either way I think additional updates here can be covered in the release/documentation issue.